### DISCUSSION TITLE
Transition scale property is not working on a live site but working perfectly locally

### DISCUSSION THREAD

**Discussion by Design-Monster2013** [tailwindlabs/tailwindcss#13634] [Help]
This is my website which I have made using tailwind css. [www.designmonsters.io](http://www.designmonsters.io/)

I am facing a weird problem. On a projects section, I have used a following property on each project:

"transition duration-500 ease-in-out hover:scale-110"

It works perfectly fine locally but on a live site the scaling property is not working.

You can view it using 'view-source'.

I have used this to all 6 projects plus I have also used it on logos slider but it is not working anywhere.

Please help me out.

I have also made 2nd site and used the same property on projects section but here scale property is working.
https://rockymountain.designmonsters.io/

Can someone also tell me how to add a smooth scrolling/animation between the sections?
is there any easy testimonial slider I can get? I have tried many but none of them works. 
I am newbie in coding field.

**Comment by wongjn** [comment]
Seems like something could be stripping the CSS rule out of the CSS. Consider checking your production configuration in case some process is unintentionally remove the CSS rule that Tailwind compiled.

**Comment by AyoubBenabid** [comment]
In case someone is having this issue, it happened for me in a WordPress website, it was because of lazy loading images, once I disabled it, the issue was solved for me
